ui/docs/reference/core/worker-manager/google.md

```diff
@@ -33,12 +33,28 @@ It is the project *ID* and not the human-readable project name.
 
 The `instancePermissions` configuration defines the permissions granted to the GCP role assumed by the workers.
 Each string in this array is a GCP IAM permission string.
-Typically, this will include permissions to write to StackDriver.
+Typically, this will include permissions to write to StackDriver, such as `logging.logEntries.create`.
 
 The GCP credentials are provided either in string form (`creds`) or in a file (`credsFile`).
-In either case, the data is the large JSON object containing a service account's keys.
+In either case, the data is the large JSON object containing a service account's keys. The object looks something like this:
 
-These credentials must carry the roles
+```
+{
+  "type": "service_account",
+  "project_id": "my-project-id-1234",
+  "private_key_id": "abc123",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nabc123\n-----END PRIVATE KEY-----\n",
+  "client_email": "thisthing@something.iam.gserviceaccount.com",
+  "client_id": "1234",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/thisthing%40something.iam.gserviceaccount.com"
+}
+```
+and will need to be included as a single string in the `creds` property.
```

Comment:
^^ I added this.  @imbstack we could probably relax that by calling `JSON.stringify` on any non-string input?  New bug?